test/unit/models/participantEndpoint/facade.test.js

```diff
@@ -32,7 +32,7 @@ const mockGetParticipant = jest.fn()
 const mockSendRequest = jest.fn()
 const mockEnums = {
   Http: {
-    Headers: { FSPIOP: { DESTINATION: 'fsp1', SOURCE: 'fsp2', SWITCH: { value: 'switch' } } },
+    Headers: { FSPIOP: { DESTINATION: 'fsp1', SOURCE: 'fsp2', SWITCH: { value: 'Hub' } } },
```

Comment:
I'd suggest adding `constants.js` file to /test folder:
```
const HUB_ID = 'Hub';

module.exports = {
   HUB_ID,
};
```
and use it all over our tests:
```
// any test file
const { HUB_ID } = require('../../constants');

... { value: HUB_ID } ...
```